modules/tests/jvm/src/test/scala/skunk/LargeResponseTest.scala

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2018-2021 by Rob Norris
+// This software is licensed under the MIT License (MIT).
+// For more information see LICENSE or https://opensource.org/licenses/MIT
+
+package test
```

Comment:
I think this should be `tests` right?  A

Reply:
Maybe this is too daring: BitVector is used to decode messages in the code above (and of course elsewhere). Maybe it would improve performance using ByteV
ector. To my knowledge the PG wire protocol's smallest unit is one byte. And ByteVector is really highly optimized. It would require a huge refactoring, though. Just my 2ct.

Reply:
There shouldn’t be a significant difference and scodec’s Codec abstraction is built on BitVector.